Dockerfile

```
@@ -34,24 +37,33 @@ chown -R root:root /opt/pseudos/;                                  \
 chmod -R +r /opt/pseudos/
 
 # Install Python packages needed for AiiDA lab.
-RUN pip install                    \
-    'aiidalab==v20.05.0b1'         \
+RUN pip install 'aiidalab==v20.05.0b1'
+
+# Installing Jupyter-related things in the root environment.
+RUN /usr/bin/pip3 install          \
     'jupyterhub==0.9.4'            \
     'jupyterlab==0.35.4'           \
-    'nbserverproxy==0.8.8'
-
-# Enable nbserverproxy extension.
-RUN jupyter serverextension enable --sys-prefix --py nbserverproxy
+    'fileupload==0.1.5'            \
+    'nbserverproxy==0.8.8'         \
+    'appmode-aiidalab==0.5.0.1'    \
+    'notebook==5.7.8'              \
+    'nglview'                      \
+    'voila'
+
+# Enable extensions.
+RUN /usr/local/bin/jupyter serverextension enable --sys-prefix --py nbserverproxy
```

Just to improve readability
```suggestion
RUN /usr/local/bin/jupyter serverextension enable --py --sys-prefix nbserverproxy
```